ui/src/screens/Talents/hooks/useTalentsReducer.ts

```diff
@@ -95,8 +91,8 @@ export default function useTalentsReducer(): [TalentsState, TalentsActions] {
   const id = match?.params.id;
 
   React.useEffect(() => {
-    axios.get<TalentsResponseData>(`${process.env.REACT_APP_API}/talents`).then((res) => {
-      updateTalents(res.data.talents);
+    axios.get<PostInterface[]>(`${process.env.REACT_APP_API}/talents/1`).then((res) => {
```

Comment:
In #93, I introduce `JobsGoWhereApiClient` which is axios underneath but allow us to just pass relative URL + its authorisation token is send whenever possible.